assets/js/translator.js

```diff
@@ -538,13 +538,13 @@ function displayPaths(paths) {
     graph.nodes = nodes;
     graph.links = [];
     Object.keys(arrows).forEach(function (src) {
-        arrows[src].forEach(function (trgt) {
-            if(arrows[trgt] && arrows[trgt].indexOf(src) !== -1) {
-                graph.links.push({'source': src, 'target': trgt, 'right': true, 'left': true});
-                arrows[trgt].splice(arrows[trgt].indexOf(src), 1);
+        arrows[src].forEach(function (dst) {
+            if(arrows[dst] && arrows[dst].indexOf(src) !== -1) {
+                graph.links.push({'source': src, 'target': dst, 'right': true, 'left': true});
```

Comment:
The `target` here (and in other similar contexts) is d3.js syntax for specifying graph links.